.gitconfig

```diff
@@ -57,6 +57,13 @@
 	# a.k.a. ‘delete merged’
 	dm = "!git branch --merged | grep -v '\\*' | xargs -n 1 git branch -d"
 
+	# Instantly clean a git repo from extraneous files in a non-blocking fashion
+	purge = "!f() { \
+		declare tmp_dir=$(mktemp -d); \
+		git clean -fdxn | sed -n -r 's/^Would remove (.*)/\\1/p' | xargs mv -t \"$tmp_dir\"; \
```

Comment:
`sed -n s'/\^Would remove \(\.\*\)/\\1/p` should fix that.

Somewhat ugly but this fixes `mv -t`:

```
... | while IFS=$'\\n' read filename; do mv \"$file\" \"$tmp_dir\"; done; \
```

Whenever a command is unwieldy towards xargs, you can always pipe to a while loop and `read`.